Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -843,14 +854,14 @@ public static void SetClientNotReady(NetworkConnection conn)
         }
 
         // default ready handler.
-        static void OnClientReadyMessage(NetworkConnection conn, ReadyMessage msg)
+        void OnClientReadyMessage(NetworkConnection conn, ReadyMessage msg)
         {
             if (LogFilter.Debug) Debug.Log("Default handler for ready message from " + conn);
             SetClientReady(conn);
         }
 
         // default remove player handler
-        static void OnRemovePlayerMessage(NetworkConnection conn, RemovePlayerMessage msg)
+        void OnRemovePlayerMessage(NetworkConnection conn, RemovePlayerMessage msg)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'msg'.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4841467)